src/fides/api/ctl/migrations/versions/15a3e7483249_make_customfielddefinition_uniqueness_.py

```diff
@@ -0,0 +1,53 @@
+"""make customfielddefinition uniqueness case insensitive
+
+Revision ID: 15a3e7483249
+Revises: e92da354691e
+Create Date: 2023-05-02 15:03:56.256982
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.exc import IntegrityError
+
+# revision identifiers, used by Alembic.
+revision = "15a3e7483249"
+down_revision = "e92da354691e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    try:
+        op.execute(
+            """ CREATE UNIQUE INDEX ix_plus_custom_field_definition_unique_lowername_resourcetype
+                ON plus_custom_field_definition
+                (lower(name), resource_type)
+            """
+        )
+    except IntegrityError as exc:
+        raise Exception(
+            f"Fides attempted to create new custom field definition unique index but got error: {exc}. "
+            f"Adjust custom field names to avoid case-insensitive name overlaps for the same resource type."
+        )
+
+    # remove unnecessary index of unused field
+    op.drop_index(
```

Comment:
this is an unused field/column altogether. removing the field/column/attribute wholesale felt out of scope for this PR, but i figured we could at least get rid of the index here as we were updating indexes on this table.